include/s3select_functions.h

```diff
@@ -58,6 +58,7 @@ enum class s3select_func_En_t {ADD,
                                VERSION,
                                CASE_WHEN_ELSE,
                                WHEN_THAN,
+                               WHEN_THEN_SIMPLE,
```

Comment:
nit
replace that with WHEN_VALUE_THEN 
simple is confusing 

better to use self-explained text when possible.